frontend/public/scripts/WebWorker.js

```diff
@@ -4,9 +4,9 @@ import {
   getRegulatoryLawTypesFromZones,
   LAWTYPES_TO_TERRITORY,
   mapToRegulatoryZone
-} from '../domain/entities/regulation'
-import { getDateMonthsBefore } from '../utils'
-import { VesselLocation, vesselSize } from '../domain/entities/vessel/vessel'
+} from '../../src/domain/entities/regulation'
```

Comment:
Pourquoi le bouger dans `scripts` ? ça le rend plus difficile à trouver je trouve

Reply:
Cf ma réponse en dessous, je pense qu'il faut en effet le laisser là où il était mais le bundler dans /public/script. La méthode "officielle" pour les WebWorker c'est d'appeler `new Worker('/path/to/worker.js')`.

Reply:
MAJ du 19/102023 :

Je resolve car c'est bundlé sans soucis.